lighthouse-core/report/html/report-styles.css

```diff
@@ -387,14 +386,20 @@
   margin-right: 0;
 }
 
-.lh-audit__title {
+.lh-audit__display-text {
   flex: 1;
 }
 
+/* Prepend display text with separator. But not in Opportunities. */
+.lh-audit-group:not(.lh-audit-group--load-opportunities) .lh-audit__display-text:not(:empty):before {
+  content: '—';
```

Comment:
per discussion in chat.. let's use whitespace on both sides of the dash.

Assuming we want to use `var(--audit-item-gap)` the we'll need some margin in here too:

```cs
  margin-right: var(--audit-item-gap);
```

Reply:
done